src/Components/OATPropertyEditor/Utils.ts

```diff
@@ -25,6 +25,14 @@ export const getModelPropertyListItemName = (name) => {
     return '';
 };
 
+export const isDisplayNameDefined = (name) => {
+    return name && typeof name === 'string' && name.length > 0
+        ? true
+        : name && Object.values(name)[0].length > 0
+        ? true
+        : false;
```

Comment:
I think we can simplify this further... I don't think ternary expressions should really be used to return true/false.

E.g. `return (name && typeof name === 'string' && name.length > 0) || (name && Object.values(name)[0].length > 0);` should be equivalent?